lib/src/client.dart

```diff
@@ -357,3 +352,9 @@ class RemoteAction<O, S> {
     }
   }
 }
+
+class GenkitStream<S, F> extends StreamView<S> {
+  F? finalResult;
```

Comment:
I'd make this non-null, and throw when called if:

* The stream hasn't completed yet.
* The stream completed with an error.

If the stream has completed successfully, the API could guarantee the availability of a non-null `F`.

This field is publicly read-write. Is that intentional? I think you want to hide the setter.

Also nit: do you need the "final" in the name?